tools/elk/lib/logformat.sc

```diff
@@ -111,6 +111,43 @@ object DirectJournalFormat extends (String => Option[LogFormat]) {
 
 }
 
+
+object TimezonesInDatacentersAreSadFormat extends (String => Option[LogFormat]) {
```

Comment:
This is the most professional name for this format that I could make up that also encompassed appropriate amounts of disdain for using time zones in your data centers.

Especially a time zone that uses DST!!! 😨 😨 😨